Cargo.toml

```diff
@@ -19,3 +19,4 @@ std = []
 
 [dependencies]
 libc = "0.2.3"
+wasi = "0.11.0"
```

Comment:
[wasi](https://github.com/bytecodealliance/wasi) is not match wasmedge wasi. 
A better way is copy and fix it.
like [this](https://github.com/second-state/wasmedge-quickjs/tree/main/src/event_loop)